homeassistant/components/freebox/binary_sensor.py

```diff
@@ -44,10 +47,105 @@ async def async_setup_entry(
         for description in RAID_SENSORS
     ]
 
+    for node in router.home_devices.values():
+        if node["category"] == FreeboxHomeCategory.PIR:
+            binary_entities.append(FreeboxPirSensor(hass, router, node))
+        elif node["category"] == FreeboxHomeCategory.DWS:
+            binary_entities.append(FreeboxDwsSensor(hass, router, node))
+
+        for endpoint in node["show_endpoints"]:
+            if (
+                endpoint["name"] == "cover"
+                and endpoint["ep_type"] == "signal"
+                and endpoint.get("value") is not None
+            ):
+                binary_entities.append(FreeboxCoverSensor(hass, router, node))
+
     if binary_entities:
         async_add_entities(binary_entities, True)
 
 
+class FreeboxBinarySensor(FreeboxHomeEntity, BinarySensorEntity):
```

Comment:
```suggestion
class FreeboxHomeBinarySensor(FreeboxHomeEntity, BinarySensorEntity):
```

I would prefer adding Home precision to know those child entities are coming from the Home API